js/datagrid.ts

```diff
@@ -350,6 +344,11 @@ export class DataGridView extends DOMWidgetView {
       },
     );
 
+    // Attaching the view's iopub callbacks functions to
+    // the data model so we can use those as an
+    // argument to model.send() function in the model class.
+    this.model._view_callbacks = this.callbacks();
```

Comment:
So it looks like now, any time a view is created, it will set the model's callbacks to the view's callbacks? So that means that any model changes will automatically be associated with the last view created. That may or may not be true, but in absence of better information, probably is the most common case. If the data change happens before any view is created, this may run into problems (i.e., if `model._view_callbacks` is undefined).